Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitSettings.cpp

```diff
@@ -63,11 +63,6 @@ static void testWebKitSettings(Test*, gconstpointer)
     Test::addLogFatalFlag(G_LOG_LEVEL_WARNING);
     ALLOW_DEPRECATED_DECLARATIONS_END
     
-    // Offline application cache is true by default.
-    g_assert_true(webkit_settings_get_enable_offline_web_application_cache(settings));
-    webkit_settings_set_enable_offline_web_application_cache(settings, FALSE);
-    g_assert_false(webkit_settings_get_enable_offline_web_application_cache(settings));
```

Comment:
Instead of removing this test, please test that the default value is FALSE, then set it to TRUE, then confirm that it still returns FALSE (because the underlying setting has been removed).

Reply:
Will do.